infra/terraform/modules/lambda_mgmt/variables.tf

```diff
@@ -0,0 +1,54 @@
+variable "lambda_function_name" {
+  description = "The default name of all resources"
+}
+
+variable "lambda_runtime" {
+  default     = "go1.x"
+  description = "Runtime language for lambda function. See: https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html#SSS-CreateFunction-request-Runtime"
+}
+
+variable "handler" {
+  description = "Entrypoint filename. For `Golang` this must match `lambda_function_name`"
+}
+
+variable "zipfile" {
+  description = "Path to zip file containing code"
+}
+
+variable "enabled" {
+  default = true
+}
+
+variable "timeout" {
+  default = 3
+}
+
+variable "schedule_expression" {
+  default = "rate(1 day)"
+}
+
+variable "source_code_hash" {}
+
+variable "env_key_1" {
```

Comment:
Now that I see what you're doing here, it may make sense to call these `env_1_name`/`env_1_value`, `env_2_name`/`env_2_value`. But sorry, I know it's a bit annoying, so you're call.

Reply:
Sure.  Ok with me.